test/distributed/test_c10d.py

```diff
@@ -1683,6 +1683,8 @@ def reduce(xs, rootRank, rootTensor):
                 torch.tensor([float(self.num_gpus * (self.num_gpus + 1) / 2)]),
                 tensors[rt])
 
+    # See https://github.com/pytorch/pytorch/issues/35689
+    @skip_if_rocm
```

Comment:
this looks spelled wrong??

Reply:
Why? (although I admit its a bit strange to have `skipIfRocm` in `common.py` and `skip_if_rock` in `common_distributed.py`) https://github.com/pytorch/pytorch/blob/2b068d10b005c1c5e6e4a33e899adae2d964a4ee/torch/testing/_internal/common_distributed.py#L96